perfmetrics/scripts/presubmit_test/pr_test/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -e
+sudo apt-get update
+echo Installing git
+sudo apt-get install git
+echo Installing go-lang
+sudo snap install go --classic
```

Comment:
Can we keep the same version of go as in https://github.com/GoogleCloudPlatform/gcsfuse/blob/b0ee644e977687d4994f90d9d3c214fdc1a353d1/tools/package_gcsfuse_docker/Dockerfile#L20 ?

Reply:
It is fetching this version only by this.

Reply:
Right but in future, it may start fetching more latest version than the one we are using.

Reply:
fixed